views/Chart.react.js

```diff
@@ -1,18 +1,44 @@
 import React from 'react'
-import Button from '@material-ui/core/Button';
+import { compose } from 'redux'
+import { withStyles } from '@material-ui/core/styles'
+import { connect } from 'react-redux'
+
+import Button from '@material-ui/core/Button'
+
+import getDefaultStyles from './fe-utils/styleUtil';
+
+import AppLayout from './layouts/AppLayout'
+
 import { routes } from './routes/routes'
 
-export default function Charts() {
+const Charts = (props) => {
+  const { user, classes } = props
+
   return (
-    <>
-      <h1> Charts</h1>
-      <Button
-        variant="outlined"
-        color="primary"
-        href={routes.newChart}
-      >
-        New Chart
-      </Button>
-    </>
+    <div className={classes.root}>
```

Comment:
can this div and class go on the `AppLayout`?

Reply:
[3c29fae](https://github.com/AMP-SCZ/dpdash/pull/87/commits/3c29fae6c41739cf5c666b5f0729b2f54fb47adb)